sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CustomFormModelStatus.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * Defines enum values for CustomFormModelStatus.
+ */
+public final class CustomFormModelStatus extends ExpandableStringEnum<CustomFormModelStatus> {
+
+    /**
+     * Static value Creating for CustomFormModelStatus.
+     */
+    public static final CustomFormModelStatus CREATING = fromString("creating");
+
+    /**
+     * Static value Creating for CustomFormModelStatus.
+     */
+    public static final CustomFormModelStatus READY = fromString("ready");
+
+    /**
+     * Static value Creating for CustomFormModelStatus.
```

Comment:
This javadoc doesn't look right for INVALID CustomFormModelStatus . Same for READY too.

Reply:
Updated.